modules/tinymce/src/core/main/ts/table/TableUtil.ts

```diff
@@ -0,0 +1,53 @@
+/**
```

Comment:
This file name really should be `TableUtils` as theres more than one utility function here.